tests/unit/s2n_session_ticket_test.c

```diff
@@ -1027,7 +1046,9 @@ int main(int argc, char **argv)
         /* Verify that the client received NST which is encrypted using a key which is at it's peak encryption */
         serialized_session_state_length = s2n_connection_get_session_length(client_conn);
         EXPECT_EQUAL(s2n_connection_get_session(client_conn, serialized_session_state, serialized_session_state_length), serialized_session_state_length);
-        EXPECT_BYTEARRAY_EQUAL(serialized_session_state + S2N_PARTIAL_SESSION_STATE_INFO_IN_BYTES, ticket_key_name2, s2n_array_len(ticket_key_name2));
+        EXPECT_BYTEARRAY_EQUAL(serialized_session_state + S2N_PARTIAL_SESSION_STATE_INFO_IN_BYTES
+                        + S2N_TICKET_VERSION_SIZE,
```

Comment:
You use this offset in so many places, is it worth naming? Like `#define S2N_SESSION_TICKET_NAME_OFFSET`? Especially since you do some NOT_EQUALs, it'd be nice to know you don't typo it anywhere.